app/models/patient.rb

```diff
@@ -742,21 +742,67 @@ def self.order_by_risk(asc: true)
   end
 
   # Check for potential duplicate records. Duplicate criteria is as follows:
-  # - matching first name, last name, sex, and DoB
+  # - matching values of first name and last name. Optionally sex and DoB can also match for more detailed messaging
```

Comment:
would like to see some unit tests for this method especially since the logic is now more complicated